toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -238,8 +238,8 @@ make-4.2.1-4.cm1.x86_64.rpm
 make-debuginfo-4.2.1-4.cm1.x86_64.rpm
 mariner-check-macros-1.0-3.cm1.noarch.rpm
 mariner-release-1.0-9.cm1.noarch.rpm
-mariner-repos-1.0-11.cm1.noarch.rpm
-mariner-repos-preview-1.0-11.cm1.noarch.rpm
+mariner-repos-1.0-12.cm1.noarch.rpm
+mariner-repos-preview-1.0-12.cm1.noarch.rpm
```

Comment:
The toolchain manifest lists all packages built as part of a toolchain build. Add in the `mariner-repos-extra` package here.